downstream/titles/aap-containerized-install/docinfo.xml

```diff
@@ -1,9 +1,9 @@
-<title>Red Hat Ansible Automation Platform Upgrade and Migration Guide</title>
+<title>Containerized Ansible Automation Platform Installation Guide</title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.4</productnumber>
-<subtitle>Upgrade and migrate legacy deployments of Ansible Automation Platform</subtitle>
+<subtitle>Containerized Ansible Automation Platform Installation Guide</subtitle>
 <abstract>
-<para>This guide shows you how to upgrade to the latest version of Ansible Automation Platform and migrate legacy virtual environments to automation execution environments.</para>
+<para>Containerized Ansible Automation Platform Installation Guide</para>
```

Comment:
Paragraph needs modification. I just changed the paragraph and subtitle when checking my local build.